### DISCUSSION TITLE
The DOTween extension asmdef seems to not set the "UNITASK_DOTWEEN_SUPPORT" automatically?

### DISCUSSION THREAD

**Discussion by TMPxyz** [Cysharp/UniTask#510] [Q&A]
I installed the DOTween from assetstore and tried to await on `transform.DOScaleX`, but the compiler complained for not found the extension methods for tween.

So I deleted the  `#if UNITASK_DOTWEEN_SUPPORT` from `DOTweenAsyncExtension.cs` and it just worked.

It seems that UniTask DOtween extension asmdef's version defines doesn't work as expected.

Does anyone see the same issue? I'm using 2023.1.

**Comment by Odaimoko** [comment]
You need to add `UNITASK_DOTWEEN_SUPPORT` in ProjectSettings  for your platform.

<img width="658" alt="image" src="https://github.com/Cysharp/UniTask/assets/7827700/e461b56e-a9c0-49d2-a2c4-f6960af520b5">
